AutomaticComponentToolkit/lib3mf.xml

```diff
@@ -1023,682 +1787,2491 @@
-			<param name="Attachment" type="handle" class="Attachment" pass="in" description="attachment that holds the texture's image information."/>
+		<method name="SetAttachment"
+			description="Sets the texture's package path to the path of the attachment.">
+			<param name="Attachment" type="handle" class="Attachment" pass="in"
+				description="attachment that holds the texture's image information." />
 		</method>
 		<method name="GetContentType" description="Retrieves a texture's content type.">
-			<param name="ContentType" type="enum" class="TextureType" pass="return" description="returns content type enum."/>
+			<param name="ContentType" type="enum" class="TextureType" pass="return"
+				description="returns content type enum." />
 		</method>
 
 		<method name="SetContentType" description="Retrieves a texture's content type.">
-			<param name="ContentType" type="enum" class="TextureType" pass="in" description="new Content Type"/>
+			<param name="ContentType" type="enum" class="TextureType" pass="in"
+				description="new Content Type" />
 		</method>
 
 		<method name="GetTileStyleUV" description="Retrieves a texture's tilestyle type.">
-			<param name="TileStyleU" type="enum" class="TextureTileStyle" pass="out" description="returns tilestyle type enum."/>
-			<param name="TileStyleV" type="enum" class="TextureTileStyle" pass="out" description="returns tilestyle type enum."/>
+			<param name="TileStyleU" type="enum" class="TextureTileStyle" pass="out"
+				description="returns tilestyle type enum." />
+			<param name="TileStyleV" type="enum" class="TextureTileStyle" pass="out"
+				description="returns tilestyle type enum." />
 		</method>
 
 		<method name="SetTileStyleUV" description="Sets a texture's tilestyle type.">
-			<param name="TileStyleU" type="enum" class="TextureTileStyle" pass="in" description="new tilestyle type enum."/>
-			<param name="TileStyleV" type="enum" class="TextureTileStyle" pass="in" description="new tilestyle type enum."/>
+			<param name="TileStyleU" type="enum" class="TextureTileStyle" pass="in"
+				description="new tilestyle type enum." />
+			<param name="TileStyleV" type="enum" class="TextureTileStyle" pass="in"
+				description="new tilestyle type enum." />
 		</method>
 
 		<method name="GetFilter" description="Retrieves a texture's filter type.">
-			<param name="Filter" type="enum" class="TextureFilter" pass="return" description="returns filter type enum."/>
+			<param name="Filter" type="enum" class="TextureFilter" pass="return"
+				description="returns filter type enum." />
 		</method>
 
 		<method name="SetFilter" description="Sets a texture's filter type.">
-			<param name="Filter" type="enum" class="TextureFilter" pass="in" description="sets new filter type enum."/>
+			<param name="Filter" type="enum" class="TextureFilter" pass="in"
+				description="sets new filter type enum." />
 		</method>
 	</class>
 
+	<enum name="ImplicitNodeType" description="The type of the node">
```

Comment:
ImplicitNodeType and other implicit enums starts from value=1 rather than value=0. Is it by purpose?